fe2-android/app/src/test/java/com/github/dedis/popstellar/utility/handler/ConsensusHandlerTest.java

```diff
@@ -74,7 +74,7 @@
   private static final ConsensusElect elect =
       new ConsensusElect(CREATION_TIME, KEY_ID, TYPE, PROPERTY, VALUE);
 
-  private static final Gson GSON = Injection.provideGson();
+  private static final Gson GSON = JsonModule.provideGson();
```

Comment:
Is there a reason that the Gson is not injected but directly use the provideGson ?

Reply:
Yes, as it is a Unit test, Hilt injection does not work for the tests. (It can probably be setup, but I have no idea how)

So I just called the provider directly.

Reply:
There is no need to use Hilt injection in unit tests.